RESUME/api_views.py

```
@@ -17,365 +20,316 @@
+        if self.error:
+            return self.error
+        try:
+            experience: Experience = self.authenticated_user.experience_set.get(
+                id=experience_id, resume_section__resume=self.resume
+            )
+            resume_sec = experience.resume_section
+            experience.delete()
+            resume_sec.delete()
+            return Response(status=status.HTTP_204_NO_CONTENT)
+        except Education.DoesNotExist:
+            return ErrorResponse(code=ErrorCode.NOT_FOUND, message=f'Experience not found!',
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)